### DISCUSSION TITLE
Time problem

### DISCUSSION THREAD

**Discussion by myxr6turbo** [Security-Onion-Solutions/securityonion#1907] [Unsupported Versions] [answered]
Hi guys,

I live in Victoria, Australia and we have daylight savings. After a fresh install and I run the date command it reads 1 hour forward. If I manually enter the time from the command line it causes problems with the logs and the data is wrong. What do I need to do?

Thanks,

**Comment by weslambert** [comment, marked as answer]
Can you let me know where it reads 1 hour forward? In the browser?  The system itself should be using UTC time.

**Reply by myxr6turbo** [reply]
If I say show me anything within the last hour nothing will show. If I say show me anything within 2 hours it will show me stuff that is 1 hour ahead of my current local time.

**Reply by jertel** [reply]
Possibly related: #1824

**Comment by myxr6turbo** [comment]
OK something strange is happening. When I built the server I did a date and it said 1 hour ahead. I did hwclock it said 1 hour ahead. When I went to alerts, ungroup to show me everything in the last hour. It was empty. I had to set it 2 hours ahead and it showed data and in the time stamp column it was 1 hour ahead of my current local clock.

I then left it over the weekend and tried to do a search today and nothing. I then ran date and its now showing 15/11/2020 21:05:59 UTC and if I run hwclock it shows 15/11/2020 09:05:59 UTC but the real date and time is 16/11/2020 8:05:00.

I have checked the ESXi server and it's showing the correct date and time.

**Reply by dougburks** [reply]
It's normal for Security Onion to set the OS timezone to UTC:
https://docs.securityonion.net/en/2.3/timezones.html?highlight=utc

**Reply by myxr6turbo** [reply]
![time-difference2](https://user-images.githubusercontent.com/29743755/99328102-833bae80-28cf-11eb-9d8b-dc2141f54b4b.jpg)
![time-difference](https://user-images.githubusercontent.com/29743755/99328111-88006280-28cf-11eb-9c90-41f5a386fe60.jpg)

Here is what I see after a brand new install
![time-difference3](https://user-images.githubusercontent.com/29743755/99328289-dc0b4700-28cf-11eb-9f80-23ceebd1fdcb.jpg)

**Reply by dougburks** [reply]
As I mentioned above, it's normal for Security Onion to set the OS timezone to UTC:
https://docs.securityonion.net/en/2.3/timezones.html?highlight=utc

And as jertel mentioned above, there is a currently a known issue with the web interface relating to time that will be fixed in the next release.
